src/sections/Planning/PlanningCardHover.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './PlanningCardHover.scss';
+
+const PlanningCardHover = ({children,className,style,setStyle,setStyleHover}) => {
+  className = 'planning-card__hover ' + className;
```

Comment:
why call it __hover when it is not always going to be hover?

Reply:
We are showing this component only on hover(mouse enter into parent)